modules/nf-core/tcoffee/alncompare/main.nf

```diff
@@ -27,8 +27,8 @@ process TCOFFEE_ALNCOMPARE {
     """
     export TEMP='./'
     t_coffee -other_pg aln_compare \
-        -al1 ${ref_msa} \
-        -al2 ${msa} \
+        -al1 <(unpigz -cdf ${ref_msa})\
```

Comment:
Maybe you should check whether the file is compressed or not before to avoid unnecessary processing

Reply:
yes, good point. pigz does autodetect this, but the detection might add some overhead compared to uncompressing based on the file extension.